modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase

```diff
@@ -2077,12 +2077,35 @@ definition {
 	}
 
 	@description = "Verify that the user can switch between Details and Revision Story Tab"
-	@ignore = "Test Stub"
 	@priority = "4"
 	test CanSwitchBetweenDetailsRevisionTabs {
+		property test.name.skip.portal.instance = "ProcessBuilderKaleoDesignerReact#CanSwitchBetweenDetailsRevisionTabs";
+
+		Workflow.openWorkflowListView();
+
+		Workflow.addNewDefinition(workflowDefinitionTitle = "New Workflow Definition");
+
+		ProcessBuilderKaleoDesignerReact.addConnector(
+			connectorIn = "end",
+			connectorLabel = "Start-End Connector",
+			connectorOut = "start",
+			key_direction = "bottom");
+
+		PortletEntry.publish();
+
+		Alert.viewSuccessMessageText(successMessage = "Workflow published successfully.");
+
+		Workflow.openWorkflowListView();
+
+		Workflow.viewDefinitionPublished(workflowDefinitionTitle = "New Workflow Definition");
+
+		Workflow.gotoEditWorkflowDefinition(workflowDefinitionTitle = "New Workflow Definition");
+
+		ProcessBuilderKaleoDesignerReact.gotoInfoWorkflowDefinition();
 
-		// TODO CanSwitchBetweenDetailsRevisionTabs pending implementation
+		Navigator.gotoNavTab(navTab = "Revision History");
 
```

Comment:
You can add a step to assert if the Revision Story tab[and the respective elements] is present